packages/@coorpacademy-components/locales/en/global.json

```diff
@@ -42,6 +48,8 @@
   "empty_search_result_description": "Try adjusting your search to find what you are looking for.",
   "empty_search_result_clear_search": "Clear search",
   "keep_subscription": "Keep my subscription",
+  "learning_priorities": "Learning priorities",
```

Comment:
can you add a new locale I've forgot to add, to avoid me to do a PR just for that, thanks:
``"learning_priorities_description": "Your top learning priorities defined by your company"``

Reply:
Sure! Added 👍